entrypoint.sh

```diff
@@ -2,16 +2,19 @@
 
 set -o pipefail
 
-python3 -m venv dbt
```

Comment:
Removed creating and entering the virtual env which didn't seem necessary. For the poetry setup, those dependencies will also be installed outside of a virtual environment so the base `dbt` functions will behave the same.

Reply:
I think that I added this initially as I was seeing some warnings suggesting a venv